packages/docusaurus/templates/classic/docs/doc1.md

```diff
@@ -15,3 +15,8 @@ I can write content using [GitHub-flavored Markdown syntax](https://github.githu
 - Hey
 - Ho
 - Let's Go
+
+
```

Comment:
Redundant new line

Reply:
Removed the live coding example because it's added on the other doc.